service_files/phosphor-wait-for-active-passive-target.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Wait for obmc-bmc-active or obmc-bmc-passive target to be started
+After=xyz.openbmc_project.State.BMC.Redundancy.service
+
+[Service]
+ExecStart=/usr/libexec/phosphor-state-manager/wait-for-active-passive-target.sh
+Type=oneshot
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
So if this service fails we'll still go to Ready (only has WantedBy). Or will we also add this service to the critical service json and cause BMC Quiesced via that?

Reply:
Hmm, I think quiescing is probably the right thing to do here, so that it doesn't just get permanently stuck in NotReady and people know to reboot if it's Quiesced.  I'll do some testing of that.